src/BuildScriptGenerator/FinalStretchVersions.cs

```diff
@@ -0,0 +1,14 @@
+// This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Common
+{
+    public static class FinalStretchVersions
+    {
+        public const string FinalStretchDotNetCore31SdkVersion = "3.1.420";
+        public const string FinalStretchNetCoreApp31RunTimeVersion = "3.1.26";
+        public const string FinalStretchAspnetCoreApp31RunTimeVersion = "3.1.26";
+        public const string FinalStretchDotNet60SdkVersion = "6.0.301";
+        public const string FinalStretchNetCoreApp60RunTimeVersion = "6.0.6";
```

Comment:
Nit: should this include a "Dot"? :)

Reply:
Similar comment above of course

Reply:
I tried to maintain the naming to be consistent with what existed in the constants.yaml, e.g. 
![image](https://user-images.githubusercontent.com/107068277/187309489-844278bf-1de8-4d81-8c81-3f0bf8560260.png)


but yeah I like this more and I need to make some changes anyways to this PR, so I'll update

Reply:
OK thanks, I didn't realize we used this somewhere else already. Likely something we forgot at the time?
Thanks for updating the name!